docker/development/medusa-be/Dockerfile

```
@@ -2,17 +2,81 @@ FROM node:22-slim AS base
 ENV PNPM_HOME="/pnpm"
 ENV PATH="$PNPM_HOME:$PATH"
 
-RUN apt update && apt install curl -y
+RUN apt-get update && apt-get install -y curl --no-install-recommends && \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)